packages/relay/src/lib/services/metricService/metricService.ts

```diff
@@ -252,6 +321,54 @@ export default class MetricService {
     });
   }
 
+  /**
+   * Initialize consensus node reset counter for duration
+   * @param {Registry} register
+   * @returns {Counter} Consensus node client reset metric
+   */
+  private initConsensusClientResetDurationMetric(register: Registry): Counter {
```

Comment:
The methods `initConsensusClientResetDurationMetric()`, `initConsensusClientResetErrorCodeMetric()`, and `initConsensusClientResetTransactionMetric()` are essentially the same. It might be better to group them into a more general method to avoid code repetition. We could place this general method in `./packages/relay/src/utils.ts`, making it reusable in other areas as well.

Reply:
Good idea! Done.